ykpersonalize.c

```diff
@@ -377,7 +377,7 @@ int main(int argc, char **argv)
 
 				if((usb_mode & 0xf) == MODE_CCID || (usb_mode & 0xf) == MODE_U2F ||
 						(usb_mode & 0xf) == MODE_U2F_CCID) {
-					fprintf(stderr, "WARNING: This tool will not be able to switch back from the selected mode, really commit? (y/n) [n]: ");
+					fprintf(stderr, "WARNING: Changing mode will requires to use another tool (ykneomgr or u2f-host) if OTP mode is disabled, really commit? (y/n) [n]: ");
```

Comment:
Typo: "will requires"
The message is also less clear than the older one because it doesn't indicate for what purpose the user will need "another tool". Old message said it was needed to "switch back from the selected mode".

I suggest something like this:
> Changing the mode will require you to use another tool (ykneomgr or u2f-host) to switch back if OTP mode is disabled, really commit? (y/n) [n]: